src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets.properties

```diff
@@ -223,6 +223,7 @@ doclet.Type=Type
 doclet.Modifier_and_Type=Modifier and Type
 doclet.Implementation=Implementation(s):
 doclet.search=SEARCH:
+doclet.search_placeholder=Search
```

Comment:
Later we should consider a better placeholder, such as an example input. We already have the adjacent "SEARCH" text label and the magnifying glass icon. Something tells me that the user has a pretty good idea what this field is for and that they don't need yet another "Search".